src/Parsing/Test/PythonVersions.cs

```diff
@@ -81,6 +81,14 @@ public static class PythonVersions {
             Python27,
             Python27_x64).FirstOrDefault() ?? NotInstalled("v2");
 
+        public static InterpreterConfiguration LatestAnaconda3X => GetVersions(
+            Anaconda36,
```

Comment:
The latest Anaconda is at Python 3.7, can that be added as an option?

Reply:
Sure